packages/admin-panel/cypress/e2e/survey/surveyImport.spec.js

```diff
@@ -12,15 +13,40 @@ import {
   selectReportingPeriod,
   selectDataService,
   importSurvey,
+  searchBySurveyName,
+  closeImportSurveyForm,
 } from '../../support';
 
-describe('import survey file', () => {
-  before(() => {
-    loginAsSuperUser();
-    openImportSurveyForm();
+beforeEach(() => {
+  loginAsSuperUser();
+  openImportSurveyForm();
+});
+
+describe('import new survey file', () => {
+  // Check error message.
+  it('Name does not match the sheet Name ', () => {
+    enterSurveyName('Test');
+    cy.uploadFile('surveys/Test Survey_1.xlsx');
+    importSurvey();
+    // assertion for error message.
+    cy.get('form').should('includes.text', 'Import failed');
```

Comment:
While checking front end messages is a great start, in order to have more confidence in our tests we should also check the state of the data after the import. Of course, this should be done using a UI accessible to the user (eg the survey search table). I guess that this is what a tester would do too: instead of just trusting the dialog message, you would verify that things work as expected by searching the data.

In this particular case, what we could do is search the survey table and confirm that the survey was not imported.

Happy for this to be added at a later stage though 🙂